presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -114,7 +114,6 @@ The following is a minimal configuration for the coordinator:
     http-server.http.port=8080
     query.max-memory=50GB
     query.max-memory-per-node=1GB
-    query.max-total-memory-per-node=2GB
```

Comment:
I would leave max-total-memory-per-node in all the the example settings in the docs.  IMO it's much better practice to set max-total-memory than it is max-memory because it corresponds more directly the actual memory the query will use.  That means it's easier to ensure you're configuring your cluster appropriately for the resources you have.  What is considered "user" vs. "system" memory is in some ways kind of arbitrary.

Reply:
I agree that it's better practice to set it explicitly, however the doc mentions this config as **minimal configuration**. That's why I have deleted optional `query.max-total-memory-per-node` since it's not required anymore.
Actually it follows `query.max-memory` and `query.max-total-memory` logic - there is no `query.max-total-memory` in minimal config.
Probably it makes sense to add a recommended config separately to the doc.
What do you think?